packages/libs/react-ui/src/components/Tabs/Tabs.css.ts

```diff
@@ -85,13 +59,89 @@ export const selectorLine = style([
   {
     width: 0,
     borderWidth: 0,
-    borderBottomWidth: tokens.kda.foundation.border.width.normal,
-    borderColor: tokens.kda.foundation.color.accent.brand.primary,
+    borderBottomWidth: token('border.width.normal'),
+    borderColor: token('color.border.tint.@focus'),
     transition: 'transform .4s ease, width .4s ease',
     transform: `translateX(0)`,
   },
 ]);
 
+export const tabItemClass = recipe({
+  base: [
+    atoms({
+      display: 'flex',
+      alignItems: 'center',
+      cursor: 'pointer',
+      paddingBlock: 'n2',
+      paddingInline: 'md',
+      fontSize: 'md',
+      fontWeight: 'secondaryFont.bold',
+      backgroundColor: 'transparent',
+      color: 'text.base.default',
+      outline: 'none',
+      borderRadius: 'xs',
+    }),
+    {
+      transition:
+        'background-color .4s ease, color .4s, border-bottom .4s ease-in-out',
+      whiteSpace: 'nowrap',
+      selectors: {
+        '&[data-selected="true"]': {
+          backgroundColor: token('color.background.base.@active'),
+          color: token('color.text.base.@active'),
+        },
+        '&[data-hovered="true"]': {
+          color: token('color.text.base.@hover'),
+        },
+      },
+    },
+  ],
+  variants: {
+    inverse: {
```

Comment:
I wonder if this was the dark/light mode things, probably not but still curious

Reply:
No this is to support a nested structure, A UX pattern is to have a lighter background color for the active tab so the inner tabs component can have the background color of the outer active tab.